### PR TITLE
docs(@angular/cli): update node versions

### PR DESCRIPTION
**Pull request by ramfattah**
Update readme documentation in regards to install the appropriate node versions as given per requirements in Angular CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ To get started locally, follow these instructions:
 
 1. If you haven't done it already, [make a fork of this repo](https://github.com/angular/angular-cli/fork).
 1. Clone to your local computer using `git`.
-1. Make sure that you have Node 12.14 or 14.0 installed. See instructions [here](https://nodejs.org/en/download/).
+1. Make sure that you have Node v12.20, v14.15, or v16.10 installed. See instructions [here](https://nodejs.org/en/download/).
```
Comment (alan-agius4) [inline README.md:52]: The versions here are actually referring to the versions that are needed to develop the Angular CLI. https://github.com/angular/angular-cli/blob/68490f1553468576c3c9a14cfefd0320f89a520c/package.json#L44

**Review by alan-agius4 — COMMENTED** [review]
Kindly sign the CLA and fix the commit message as per commit message guidelines.
https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#commit

File: README.md (hunk 1 above)
Comment (ramfattah) [reply README.md:52]: Interestingly, before I had node v 16.5.0. I ran `ng serve` on the root directory of the Angular project, and the CLI pointed out that need these node versions. After installing node v 16.10, I was successfully able to serve the ng app.

Comment (alan-agius4) [reply README.md:52]: The versions of Node.js that the Angular CLI requires can be found here https://angular.io/guide/setup-local#prerequisites

**Comment by alan-agius4** [comment]
@ramfattah, kindly update the commit message and not just the PR title.

**Comment by alan-agius4** [comment]
Thanks for your contribution, but this has been superseded by https://github.com/angular/angular-cli/pull/22587.

**Pull request closed without merging**